data/unpacked/latest.json

```diff
@@ -10537,15 +10534,15 @@
     {
       "name": "America/Cayman",
       "abbrs": [
-        "KMT",
+        "CMT",
```

Comment:
This changed is caused by America/Cayman now being a link to America/Panama
https://github.com/eggert/tz/commit/72a3ac9cb075b934c13885b6ef2616374b704fe6#diff-9c77a7aa95323ac0135649098471287eL2600